text/survey_analysis_report2.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "A folder structure to organise files in a reserch project"
+bibliography:  "GIN-tonic-paper-zotero.bib"
+author:
+  - Julien Colomb
+  - The Gin-tonic team
+output: 
+  bookdown::html_document2:
+    keep_md: true
+
+---
+
+
+![titleimage: Some positive feedback received in the survey.](../50_figures/pulication_ready_figures/blogtitle.jpeg)
+
+# Summary
+
+In contrast to software developers, researchers have no standard way to organise their files.
```

Comment:
In contrast to software developers, researchers mostly do not organize their files according to some common standard.